bin/measureHscRC2Metrics.sh

```diff
@@ -2,26 +2,35 @@
 
 set -e
 
-pipetask --long-log run -j $NUMPROC -b ${RC2_SUBSET_DIR}/SMALL_HSC/butler.yaml -p ${RC2_SUBSET_DIR}'/pipelines/DRP.yaml#nightlyStep1' -i HSC/RC2/defaults --register-dataset-types -o jenkins/step1
+REPO=${RC2_SUBSET_DIR}/SMALL_HSC
+OUTPUT_COLLECTION_STUB="jenkins"
+RC2_SUBSET_PIPELINE=${DRP_PIPE_DIR}/pipelines/HSC/DRP-RC2_subset.yaml
 
-echo "Running step2a on all visits"
-pipetask --long-log run -j $NUMPROC -b ${RC2_SUBSET_DIR}/SMALL_HSC/butler.yaml -p ${RC2_SUBSET_DIR}'/pipelines/DRP.yaml#nightlyStep2a' -i jenkins/step1 --register-dataset-types -o jenkins/step2
+echo "Running nightlyStep1 on all visits"
+pipetask --long-log run --register-dataset-types -j $NUMPROC -b $REPO -i HSC/RC2/defaults -o $OUTPUT_COLLECTION_STUB/step1 -p $RC2_SUBSET_PIPELINE#nightlyStep1
 
-echo "Running step2b on tract 9813"
-pipetask --long-log run -j $NUMPROC -b ${RC2_SUBSET_DIR}/SMALL_HSC/butler.yaml -p ${RC2_SUBSET_DIR}'/pipelines/DRP.yaml#nightlyStep2b'  -d "tract = 9813 and skymap = 'hsc_rings_v1'" --register-dataset-types -o jenkins/step2
+echo "Running nightlyStep2a on all visits"
+pipetask --long-log run --register-dataset-types -j $NUMPROC -b $REPO -i $OUTPUT_COLLECTION_STUB/step1 -o $OUTPUT_COLLECTION_STUB/step2 -p $RC2_SUBSET_PIPELINE#nightlyStep2a
 
-echo "Running step2c without multiprocessing"
-pipetask --long-log run -b ${RC2_SUBSET_DIR}/SMALL_HSC/butler.yaml -p ${RC2_SUBSET_DIR}'/pipelines/DRP.yaml#nightlyStep2c' --register-dataset-types -o jenkins/step2
+echo "Running nightlyStep2b on tract 9813"
+pipetask --long-log run --register-dataset-types -j $NUMPROC -b $REPO -o $OUTPUT_COLLECTION_STUB/step2 -p $RC2_SUBSET_PIPELINE#nightlyStep2b -d "skymap = 'hsc_rings_v1' AND tract = 9813"
 
-echo "Running step2d on all visits"
-pipetask --long-log run -j $NUMPROC -b ${RC2_SUBSET_DIR}/SMALL_HSC/butler.yaml -p ${RC2_SUBSET_DIR}'/pipelines/DRP.yaml#nightlyStep2d' --register-dataset-types -o jenkins/step2
+echo "Running nightlyStep2c without multiprocessing"
+pipetask --long-log run --register-dataset-types -j $NUMPROC -b $REPO -o $OUTPUT_COLLECTION_STUB/step2 -p $RC2_SUBSET_PIPELINE#nightlyStep2c
```

Comment:
Should remove "-j $NUMPROC", as this step is meant to skip multiprocessing.

Reply:
Goodness, thanks for catching this Jeff! My test run also failed with the error:
```
lsst.ctrl.mpexec.mpGraphExecutor.MPGraphExecutorError: Task lsst.fgcmcal.fgcmBuildStarsTable.FgcmBuildStarsTableTask does not support multiprocessing; use single process
```
which confirms the necessity for this! Thanks again!